UnityProject/Assets/Scripts/Health/BloodHealth/BloodSystem.cs

```diff
@@ -197,7 +197,7 @@ private void LoseBlood(float amount)
 		}
 
 
-		EffectsFactory.BloodSplat(transform.position, scaleOfTragedy, bloodSplatColor);
+		//EffectsFactory.BloodSplat(transform.position, scaleOfTragedy, bloodSplatColor);
```

Comment:
Got plans for this commented out code? Commented code exists in a few other files too.

Reply:
Stick a TODO comment on them

Reply:
Ill add todos, they are going to be in a follow up PR but will still add TODOs